common/Services/IExperimentationService.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using DevHome.Common.Models;
+
+namespace DevHome.Common.Services;
+
+public interface IExperimentationService
+{
+    bool IsEnabled(string key);
+
+    void SetIsEnabled(string key, bool value);

```

Comment:
To make this more idiot-proof (I'm the idiot, I had to think about it for a second) I wonder if these should be named `IsFeatureEnabled` and `SetIsFeatureEnabled`. For a second I thought maybe the whole ExperimentationService was enabled or disabled.

Reply:
Fixed!